config_spec.toml

```diff
@@ -56,3 +56,9 @@ name = "grpc_port"
 type = "u16"
 optional = true
 doc = "The port the grpc server will run on. Defaults to 7000."
+
+[[param]]
+name = "skip_version_check"
+type = "bool"
+default = "false"
+doc = "Skip checking the LND version. Otherwise, LNDK checks that the LND version and build tags are compatible with LNDK."
```

Comment:
We can remove the mention of the build tags here since we're now checking them whether or not this is set.